src/main/java/com/amazonaws/encryptionsdk/internal/FrameDecryptionHandler.java

```diff
@@ -119,11 +119,6 @@ public ProcessingSummary processBytes(final byte[] in, final int off, final int
             if (currentFrameHeaders_ == null) {
                 currentFrameHeaders_ = new CipherFrameHeaders();
                 currentFrameHeaders_.setNonceLength(nonceLen_);
-                if (frameSize_ == 0) {
-                    // if frame size in ciphertext headers is 0, the frame size
-                    // will need to be parsed in individual frame headers.
-                    currentFrameHeaders_.includeFrameSize(true);
-                }
```

Comment:
Why are we removing this logic?

Reply:
After discussion, I'll revert this part of the change and have created a separate issue to track removing this code (which was a partially implemented variable frame length feature): https://github.com/aws/aws-encryption-sdk-java/issues/167